signingscript/docker.d/passwords.yml

```diff
@@ -85,7 +86,8 @@ in:
             - ["https://prod.autograph.prod.webservices.mozgcp.net",
                {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_USERNAME"},
                {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_PASSWORD"},
-               ["gcp_prod_autograph_authenticode_202404", "gcp_prod_autograph_authenticode_202404_stub"],
+               ["gcp_prod_autograph_authenticode_202404", "gcp_prod_autograph_authenticode_202404_stub",
```

Comment:
Having `gcp_prod_autograph_authenticode_202404` here might actually not be a good idea, since the prod environment won't have that key?

Reply:
Yeah, that's a good call. Will do.